src/common/file_system/local_file_system.cpp

```diff
@@ -267,15 +267,26 @@ static std::unordered_set<std::string> getDatabaseFileSet(const std::string& pat
     return result;
 }
 
-static bool isExtensionFile(const std::filesystem::path& path) {
-    return path.extension() == extension::ExtensionUtils::EXTENSION_FILE_SUFFIX;
+static bool isExtensionFile(const main::ClientContext* context, const std::string& path) {
+    if (context == nullptr) {
+        return false;
```

Comment:
Should this silently return false instead of throwing an error?

Reply:
We should throw, we are not expected to delete any files/directories outside of the db path and extension directory